### ISSUE TITLE
Overlay of task tiles onto campaign - Possible to open tasking manager from these?

### ISSUE THREAD

**Issue by DaCor1** [hotosm/field-campaigner#384]
The title says it all. Right now the tiles from a task in the tasking manager can be overlaid on top of a campaign....to what end? I don't see them having any functionality yet (is something planned)

It would be great if, when I click on a tile, it gives me the option to jump to another window with that tile opening on the tasking manager. Note, I said option. It might be irritating if it did it automatically when a user clicked on a tile

**Comment by PaulUithol** [comment]
Yeah, I'd definitely like to see further integration with the TM - including pulling/pushing tasks (and collections of those/portfolios)
